### DISCUSSION TITLE
Possibility to roll back upgrade to previous version of Coolify

### DISCUSSION THREAD

**Discussion by sscarduzio** [coollabsio/coolify#2294] [Feature Requests]
It's already the [second time](https://github.com/coollabsio/coolify/issues/2292) a regression kills our development workflow. 

Auto updates are fine, stable vs unstable channel is fine, but the real missing feature is the ability to downgrade on an emergency situation due to a regression.

**Comment by andrasbacsai** [comment]
Atm, this is the way: https://coolify.io/docs/downgrade

**Reply by sscarduzio** [reply]
I'm using the SaaS :/

**Reply by sscarduzio** [reply]
which makes me think this is a QA issue, unfortunately.

**Comment by ton77v** [comment]
Yeah such a feature would be nice to have for Coolify Cloud.

**Reply by sscarduzio** [reply]
Hmm.. Not sure it can even be done at the moment because AFAIK Coolify SaaS is a single multi tenant instance, deployed once for all its users.
